### PR TITLE
Downgraded to play 1.3.0

### PR DESCRIPTION
**Pull request by johanneswuerbach**
The play buildpack only works for play 1.x apps. Play 2.0 apps are using the scala buildpack

https://github.com/heroku/heroku-buildpack-play#heroku-buildpack-play

> Note: This buildpack only applies to Play 1.2.x apps. Play 2.x apps are handled by the Scala buildpack

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -1,69 +1,228 @@
 # This is the main configuration file for the application.
 # ~~~~~
+application.name=example-play
+
+# Application mode
```
Comment (bacongobbler) [inline conf/application.conf:5]: let's keep this as minimal as possible. Could you please remove the comments?

File: conf/application.conf [hunk 2]
```diff
@@ -1,69 +1,228 @@
...
+# If you need to set proxy params for WS requests
+# http.proxyHost = localhost
+# http.proxyPort = 3128
+# http.proxyUser = jojo
+# http.proxyPassword = jojo
+
+# Mail configuration
+# ~~~~~ 
+# Default is to use a mock Mailer
+mail.smtp=mock
```
Comment (bacongobbler) [inline conf/application.conf:46]: this appears to be the default so we may not need this

File: conf/messages [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# You can specialize this file for each language.
```
Comment (bacongobbler) [inline conf/messages]: this is not required

File: test/data.yml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# you describe your data using the YAML notation here
```
Comment (bacongobbler) [inline test/data.yml]: not required

File: test/BasicTest.java [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import org.junit.*;
```
Comment (bacongobbler) [inline test/BasicTest.java]: not required

**Comment by johanneswuerbach** [comment]
This PR was done by deleting all existing files, generating a new app using `play new example-play`, adding the Procfile and modifying the controller. I didn't change any configs etc.

Do you want to remove all unnecessary files or should we leave the app like a default app to simplify any further upgrades?

**Comment by bacongobbler** [comment]
Personally I'd rather strip away any unnecessary files and just use the bare minimum. There won't be any upgrades happening on this app once it's working with the play buildpack. :)

**Comment by johanneswuerbach** [comment]
I removed all unnecessary files and config sections where no value is set. I left all config section, where a value is configured. Keeping those comments improves config flags clarity for me and might simplify debugging BP related errors in the future.

**Comment by mboersma** [comment]
``` console
$ git push deis master
Counting objects: 67, done.
Delta compression using up to 4 threads.
Compressing objects: 100% (57/57), done.
Writing objects: 100% (67/67), 46.73 KiB | 0 bytes/s, done.
Total 67 (delta 14), reused 0 (delta 0)
-----> Play! app detected
-----> Installing OpenJDK 1.6... done
-----> Installing Play! 1.3.0.....
...
Successfully built 5ecf9814b482
-----> Pushing image to private registry

-----> Launching... 
       done, onside-quadrant:v2 deployed to Deis
```

LGTM. Thanks @johanneswuerbach.

**Pull request merged**